clang/lib/DPCT/DPCT.cpp

```diff
@@ -1130,6 +1132,7 @@ int runDPCT(int argc, const char **argv) {
     static const std::string StartStr{"// Start"};
     static const std::string EndStr{"// End"};
     bool Flag = false;
+    llvm::outs() << llvm::raw_ostream::BLUE;
```

Comment:
if the output color is controlled by here, then don't try to embed raw string with color control

Reply:
fixed